docs/gallery/domain/ecephys.py

```diff
@@ -73,10 +71,9 @@
 #     :alt: electrodes table UML diagram
 #     :align: center
 #
-# Before creating an :py:class:`~pynwb.ecephys.ElectrodeGroup`, you need to provide some information about the
-# device that was used to record from the electrode. This is done by creating a :py:class:`~pynwb.device.Device`
-# object using the instance method :py:meth:`~pynwb.file.NWBFile.create_device`.
-
+# The electrodes table references a required :py:class:`~pynwb.ecephys.ElectrodeGroup`, which is used to represent a
+# group of electrodes. Before creating an :py:class:`~pynwb.ecephys.ElectrodeGroup`, you must define a
+# :py:class:`~pynwb.device.Device` object using the method :py:meth:`.NWBFile.create_device`.
```

Comment:
Why does this work? :py:meth:&#96;NWBFile.create_device&#96;

Reply:
I don't know, but this is the way I have figured out to display method links in the way that I want